Source/Core/DiscIO/Enums.cpp

```diff
@@ -182,6 +182,8 @@ std::string GetSysMenuVersionString(u16 title_version)
   case Region::NTSC_K:
     region_letter = "K";
     break;
+  case Region::UNKNOWN_REGION:
```

Comment:
Fair point, but it forces users to say that Dolphin is broken and they'll be able to provide us the appropriate log if we don't handle a case. A region string like "(unknown region)" might be more straight forward than just leaving the question mark (that can be misinterpreted as a question, you're right).